clients/tests/simple_test.cpp

```diff
@@ -645,3 +708,29 @@ TEST(hipfftTest, OutplaceOnly)
     fftw_destroy_plan(ref_p);
     fftw_free(ref_out);
 }
+
+TEST(hipfftTest, GetVersion)
+{
+    // valid use case(s)
+    int tmp;
+    EXPECT_EQ(hipfftGetVersion(&tmp), HIPFFT_SUCCESS);
+    // invalid use case(s)
+    // FIXME: enable once fixed (this segfaults for now)
```

Comment:
This should be trivial to fix, so do you want to just fix hipFFT and enable this part of the tests?